lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -61,6 +61,7 @@ def self.canonicalize_url(driver_url, config)
 
     # Load balancer methods
     def allocate_load_balancer(action_handler, lb_spec, lb_options, machine_specs)
+      lb_options ||= {}
```

Comment:
Hmm, depends if machine_specs can ever not be passed.  I'll take a look at the caller and see if they can simply be persuaded to pass `{}` themselves.